server/routes/calendar.test.js

```diff
@@ -650,11 +643,11 @@ beforeEach(() => {
   app = appSetup(calendarRoute, {
     calendarService,
     calendarOvertimeService,
-    DEPRECATEnotificationService,
+    notificationService,
```

Comment:
I prefer methods staying with their service as I think it's more descriptive. We're also mid service switch over, so I think it's clearer if we specify that we're calling the new service rather than the deprecated service